invenio_github/api.py

```diff
@@ -209,11 +209,11 @@ def sync_repo_hook(self, repo_id):
                 pass
 
     def check_sync(self):
-        """Check and sync if over a day has passed since the last sync."""
+        """Check if sync is required based on last sync date."""
         now = utcnow()
-        yesterday = now - timedelta(days=1)
+        expiration = now - current_app.config['GITHUB_REFRESH_TIMEDELTA']
```

Comment:
It would be nice to have some utility to handle this calculation. But there are many things that should be "standarized", like the refresh timedelta value (negative/positive), the field in `extra_data` that would hold the last sync date ("last_sync"/"updated"), etc...